.github/workflows/npc_db_validation.yml

```
@@ -0,0 +1,84 @@
+      - 'db/**'
+      - 'npc/**'
+
+jobs:
+  build:
+    # Github Actions checks for '[ci skip]', '[skip ci]', '[no ci]', '[skip actions]', or '[actions skip]' but not a hyphenated version.
+    # It's a catch-all incase a Pull Request has been opened and someone is on auto-pilot.
+    if: "!contains(github.event.head_commit.message, 'ci-skip')"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+          # The ubuntu-latest label currently points to ubuntu-18.04.
+          # Available: ubuntu-20.04, ubuntu-18.04	
+          os: [ubuntu-latest]
+          # Only a single version of GCC is required for validating NPC scripts and database changes.
+          gcc: ['10']
+          # We run build checks for both Renewal and PRE-Renewal
+          mode: ['PRE', 'RE']
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      # A simple 'yes' and 'no' can be confusing, so we use names to display in the current job then convert them for use in the compiler.
+      - name: Variable Parsing - PRE
+        if: ${{ matrix.mode == 'PRE' }} 
+        run: |
+          echo "PRERE=yes" >> $GITHUB_ENV
+      - name: Variable Parsing - RE
+        if: ${{ matrix.mode == 'RE' }} 
+        run: |
+          echo "PRERE=no" >> $GITHUB_ENV
+
+      - name: Update & Install packages
+        # Ubuntu runners already have most of the packages rAthena requires to build.
+        # https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md
+        run: |
+          sudo apt update
+          sudo apt install zlib1g-dev libpcre3-dev gcc-${{ matrix.gcc }} g++-${{ matrix.gcc }}
+          
+      - name: Start MySQL
+        run: sudo systemctl start mysql.service
+
+      - name: Setup Database
+        env:
+          DB_ROOT: 'root'
+          DB_HOST: '127.0.0.1'
+          DB_RAG: 'ragnarok'
+          # MySQL (v8.0.26) provided by Github Actions is pre-configured to use root as the password.
+          # We default to 'ragnarok' for database, user and password, so no need for additional variables in this script.
+        run: |
+          mysql -u $DB_ROOT -p$DB_ROOT -e "CREATE DATABASE $DB_RAG;"
+          mysql -u $DB_ROOT -p$DB_ROOT -e "CREATE USER '$DB_RAG'@'$DB_RAG' IDENTIFIED BY '$DB_RAG';"
+          mysql -u $DB_ROOT -p$DB_ROOT -e "GRANT SELECT,INSERT,UPDATE,DELETE,CREATE ON $DB_RAG.* TO '$DB_RAG'@'$DB_RAG';"
+      
+      - name: Create MySQL User and Setup Tables
+        run: ./tools/ci/import_sql.sh
+          
+      - name: Command - configure
+        env:
+            CONFIGURE_FLAGS: 'CC=gcc-${{ matrix.gcc }} CXX=g++-${{ matrix.gcc }} --enable-prere=${{ env.PRERE }} --enable-buildbot=yes'
+        run: ./configure $CONFIGURE_FLAGS
+      
+      # npc.sh enables all NPC scripts in the custom and test folders.
+      - name: Enable All NPCs for Testing
+        run: ./tools/ci/npc.sh
+
+      - name: Command - make clean
+        run: make clean
+
+      - name: Command - make map server
```

```suggestion
      - name: Command - make map
```
Sorry, realized you wrote the command as is in the name.

---

How did that get there...